.github/workflows/backup-database.yml

```
@@ -58,6 +58,7 @@ jobs:
 
   restore-staging:
     name: Restore de la BDD staging avec le backup le plus récent
+    if: false # Temporary: disable staging restore during pentest period
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)